### ISSUE TITLE
Question to change google chrome icon

### ISSUE THREAD

**Issue by mrcodedev** [bil-elmoussaoui/Hardcode-Tray#179]
Hello,

I'm very noob at this and don't know how it really works. What I'm trying to change is the icon of google chrome bar notifications. To change it not that I really do, because I'm using icons Ultra-Flat and not if I create a folder, or that I must do. You be obliged if you throw me a hand.

Thank you very much

**Comment by bil-elmoussaoui** [comment]
Please take a look at the documentation https://github.com/bil-elmoussaoui/Hardcode-Tray/wiki everything is well explained there! if you need any more informations feel free to ask!

**Comment by mrcodedev** [comment]
Hello,

if I followed the steps but don't exactly understand this:

When you execute the command, you are looking at the theme folder you currently have?, and if so how should be called in this case to change the icon of Google Chrome ?.

If I understand accomplish using your script I will help to make a tutorial in Spanish.

Thank you very much

**Comment by mrcodedev** [comment]
Hello again,

in the end I succeeded but your script ... is not how it works. I've been reading comments and I have seen that in the folder tmp temporary files are stored. I saw it was there the Chrome icon and I have changed.

But out of curiosity I would like if you please how it works hardcode-tray, and not if does the same thing I've done automatically or look at the folder icons. I would like to see an example of how it would work for chrome icon, to get an idea of performance.

Thank you very much

**Comment by mrcodedev** [comment]
Goodnight,

I've gotten it to work by adding -t Numix. If you would like on other issues what references look at the script to get you out icons, for example Ultra-Flat?.

Greetings and sorry for the inconvenience

**Comment by bil-elmoussaoui** [comment]
Sorry for the delay,
Well hardcode-tray will look in the database file(db.csv) and check which application is installed or not. Once this is done, the list of applications has a list of icons under /database/applicationame with the icon name, the symlink icon name and other informations. The script will verify if the icon name exists in the current theme you're using 
For example https://github.com/bil-elmoussaoui/Hardcode-Tray/blob/master/database/google-chrome#L5 This is the google chrome database file. The 8002 is the icon name in the binary file resources.pak; it's replaced by google-chrome-tray if this file exists in your theme. So if you want to add support to chrome tray icon, you just need to create this icon in your current theme /usr/share/icons/themeName/. You will need both sizes (22, 24) of the icon! As the script will detect your distro and ur desktop env and use the right icon size for you so everything will look perfect :) 
![capture du 2016-10-12 12-34-42](https://cloud.githubusercontent.com/assets/7660997/19306930/45539ec8-9078-11e6-8e04-29ac6a461df0.png)

**Comment by mrcodedev** [comment]
Wow!!!, thank you very much, I was watching it and I figured it was something. When you have a little more time I'll look thoroughly for use with other apps that aren't now supported. If you need a spanish translation and let me know if I can help in some way how little is too.
Greetings bil-elmoussaoui

**Comment by bil-elmoussaoui** [comment]
Will do think about adding translation support later( once i've fixed all the issues..) if you have any application that is not supported yet, just open a new issue! will try to fix it as soon as possible. Thanks
